src/components/Footer/Footer.js

```diff
@@ -2,7 +2,78 @@ import React from 'react';
 import './Footer.scss';
 
 const Footer = () => {
-  return <div>Footer 입니다.</div>;
+  return (
+    <div className="footer">
+      <div className="footerContents">
+        <div className="logoImg">
+          {/* 나중에 무료이미지 찾아서 넣을 때를 대비해 주석을 삭제하지 않았습니다. */}
```

Comment:
```suggestion
          {/*TODO: 나중에 무료이미지 찾아서 넣을 때를 대비해 주석을 삭제하지 않았습니다. */}
```
이렇게 나중에 뭔가 해야 하는거라면, TODO 를 prefix로 붙여주면 나중에 주석만 찾을때 쉽게 찾을 수 있습니다